runtime/src/accounts.rs

```diff
@@ -144,6 +144,8 @@ impl Accounts {
             let mut payer_index = None;
             let mut tx_rent: TransactionRent = 0;
             let mut accounts = Vec::with_capacity(message.account_keys.len());
+            let fix_rent = feature_set.rent_fix_enabled();
```

Comment:
nit: name the var the same for easy future searching:
```suggestion
            let rent_fix_enabled = feature_set.rent_fix_enabled();
```

Reply:
done d258182d236426ff5e3382433d1ab0e3fefda705